draft-ietf-tls-tls13.md

```diff
@@ -2889,9 +2887,37 @@ for each scenario:
 | Client | ClientHello ... ServerFinished     | client_handshake_traffic_secret |
 | Post-Handshake | ClientHello ... ClientFinished + CertificateRequest | client_traffic_secret_N |
 
-In all cases, the handshake context is formed by concatenating the
-indicated handshake messages, including the handshake message type
-and length fields, but not including record layer headers.
+
+### The Transcript Hash
+
+Many of the cryptographic computations in TLS make use of a transcript
+hash. This value is computed by hashing the concatenation of the
+hashes of each included handshake message, including the handshake
```

Comment:
hash of the concatenation of hashes isn't right, it's just the hash of the concatenated handshake messages